lms/templates/courseware/courseware-chromeless.html

```diff
@@ -53,7 +56,7 @@
 
   <%static:js group='courseware'/>
 
-  <%include file="../discussion/_js_body_dependencies.html" />
+  <%include file="/mathjax_include.html" args="disable_fast_preview=True"/>
```

Comment:
So _js_body_dependencies.html has disable_fast_preview set to True, and this is just keeping the same behavior (which fixes the Lettuce flaky test failure)?